scripts/irods/test/test_iquest.py

```diff
@@ -159,6 +159,11 @@ def test_iquest_does_not_fail_to_find_data_objects_with_SQL_keyword_names_select
                 self.admin.assert_icommand(['iquest', f"SELECT DATA_NAME, DATA_CREATE_TIME WHERE DATA_NAME = '{data_object}'"], 'STDOUT', expected_output)
                 self.admin.assert_icommand(['iquest', f"SELECT DATA_NAME, DATA_CREATE_TIME WHERE DATA_NAME like '%{data_object}%'"], 'STDOUT', expected_output)
 
+    def test_iquest_prints_acls_when_data_object_contains_single_quotation_mark_in_name__issue_7164(self):
```

Comment:
One point of confusion here... This is in `test_iquest` and the test name refers to `iquest`, but we appear to be testing `ils`. I understand that `ils` is just using GenQuery underneath, but shouldn't this rather be in `test_ils` since the test and #7164 are about `ils`?

Reply:
Nice catch. Yes, the test should be in test_ils.py.

Will update.

Reply:
Done.